lib/src/utils/implementations/bundle_tag_parser_js.dart

```diff
@@ -0,0 +1,62 @@
+@JS()
+library tagparser;
+
+import 'dart:typed_data';
+
+import 'package:arweave/src/models/tag.dart';
+import 'package:arweave/src/utils.dart';
+import 'package:js/js.dart';
+
+@JS()
+external Uint8List serializeTagsToBuffer(var bundleTags);
+
+@JS()
+external List<BundleTag> deserializeTagsFromBuffer(var buffer);
+
+Uint8List serializeTags({required List<Tag> tags}) {
+  final decodedTags = <Tag>[];
+  tags.forEach((tag) {
+    decodedTags.add(Tag(
+      decodeBase64ToString(tag.name),
+      decodeBase64ToString(tag.value),
+    ));
+  });
+  final data = serializeTagsToBuffer(decodedTags
+      .map((tag) => BundleTag(name: tag.name, value: tag.value))
+      .toList());
+  return data;
+}
+
+List<Tag> deserializeTags({var buffer}) {
+  try {
+    final tags = deserializeTagsFromBuffer(buffer);
+    final decodedTags = <Tag>[];
+    for (var tag in tags) {
+      decodedTags.add(Tag(
+        encodeBytesToBase64(
+            tag.name.split(',').map((e) => int.parse(e)).toList()),

```

Comment:
Isn't this already a `List`? Or maybe the `List` and `Array` are two different types?

Reply:
They are different types. In dart there's Iterables and Lists.